src/apps/rate_limiter/rate_limiter.lua

```diff
@@ -25,12 +25,15 @@ local PACKET_SIZE = 60
 
 function RateLimiter:new (arg)
    local conf = arg and config.parse_app_arg(arg) or {}
-   assert(conf.rate)
-   assert(conf.bucket_capacity)
+   --- By default, limit to 10 Mbps, just to have a default.
+   conf.rate = conf.rate or (10e6 / 8)
+   -- By default, allow for 255 standard packets in the queue.
+   conf.bucket_capacity = conf.bucket_capacity or (255 * 1500)
```

Comment:
“Just to have a default” is not an argument for having a default. I don't see a case where a default is useful, and not having an error when accidentally failing to supply `rate` and `bucket_capacity` is dangerous. Please revert this.